x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/trusted_app_card/index.tsx

```diff
@@ -75,6 +83,11 @@ export const TrustedAppCard = memo(({ trustedApp, onDelete }: TrustedAppCardProp
         }
       />
       <ItemDetailsPropertySummary name={PROPERTY_TITLES.created_by} value={trustedApp.created_by} />
+      <ItemDetailsPropertySummary
+        name={PROPERTY_TITLES.description}
+        value={trimTextOverflow(trustedApp.description || '', 100)}
```

Comment:
You should consider storing the `value` with `useMemo()`.

Reply:
Fixed.